scripts/packages/chocolatey/build.ps1

```diff
@@ -0,0 +1,45 @@
+param(
+  [string] $version = "0.3.1",
+  [switch] $isRelease
+)
+
+$tvVersion = $version
+$tvFilename = "bazel_$($version)_windows_x86_64.zip"
+if ($isRelease) {
+  $tvUri = "https://github.com/bazelbuild/bazel/releases/download/$($version)/$($tvFilename)"
+} else {
+  $tvUri = "http://localhost:8000/$($tvFilename)"
+}
+write-host "download uri: $($tvUri)"
+
+rm -force ./*.nupkg
+rm -force ./*.zip
+rm -force ./bazel.nuspec
+rm -force ./tools/chocolateyinstall.ps1
+
+Add-Type -A System.IO.Compression.FileSystem
+$zipDir = "$pwd/../../../output"
+$zipFile = "$pwd/$($tvFilename)"
+write-host "Creating zip package from directory: $zipDir to file: $zipFile"
+[IO.Compression.ZipFile]::CreateFromDirectory($zipDir, $zipFile)
```

Comment:
Let's not grab all the files from the output (who knows what is there). Instead, grab the specific files we need (I beleive that is just bazel.exe)